docs/guide/images.md

```diff
@@ -398,3 +398,171 @@ For an ImageAsset Field to be reused you have to publish it. Only then is notifi
 After establishing media library you can create object Relations between the main Content item and the image Content item being used by it.
 
 To learn more about ImageAsset Field Type and its customization see [Field Type Reference](../api/field_type_reference.md#imageasset-field-type).
+
+## Handling SVG images
+
+Currently, Ibexa DXP does not allow you to store SVG images using `Image` or `ImageAsset` FieldType. Until the full support for this MIME type is in place, you can work things around by relying on `File` FieldType and implementing custom extension which would enable you to display/download files in your templates.
+
+First, you add a proper rule in the `config/routes.yaml` file:
+
+```yaml
+app.svg_download:
+    path: /asset/download/{contentId}/{fieldIdentifier}/{filename}
+    defaults: { _controller: app.controller.content.svg:downloadSvgAction }
+```
+
+It points to a custom controller that handles the downloading of the SVG file. 
+The controller's definition (that you place in the `config/services.yaml` file) and implementation are as follows:
+
+```yaml
+App\Controller\SvgController:
+    public: true
+    arguments:
+        - "@ezpublish.api.service.content"
+        - "@ezpublish.fieldType.ezbinaryfile.io_service"
+        - "@ezpublish.translation_helper"
```

Comment:
It might not be clear that it needs to be placed under `services:`, instead of the top level.